javascript/javascript1/week3/homework4.js

```diff
@@ -0,0 +1,74 @@
+// Smart-ease - Back to the basics!
+// NOnoN0nOYes (Note taking app)
+
+// The saveNote function should push an object to the notes array with the keys content and id.
+
```

Comment:
Good practice with adding comments up in the files. Next step would be to name the files in a way that indicates what code do they contain. Eg. `notes.js`, `smartphone-usage.js`, `item-array-removal.js`, `series-duration-life-calculator.js`, `travel-time-calculator.js`...